spec/controllers/users/two_factor_authentication_controller_spec.rb

```diff
@@ -324,6 +324,13 @@ def index
                                   { otp_delivery_preference: 'sms', resend: 'true' } }
       end
 
+      it 'tracks the verification attempt event' do
+        expect_any_instance_of(IrsAttemptsApi::Tracker).to receive(:mfa_phone_verification_otp_sent)
+
+        get :send_code, params: { otp_delivery_selection_form:
+          { otp_delivery_preference: 'sms' } }
```

Comment:
style nite, I would not leave the key hanging on the previous line like that
```suggestion
        get :send_code, params: {
          otp_delivery_selection_form: { otp_delivery_preference: 'sms' }
        }
```

Reply:
Thank you! Agreed!